### ISSUE TITLE
Query problem with mariadb-10.5-0.8 and mariadb-10.5-0.15

### ISSUE THREAD

**Issue by MrCrayon** [cytopia/devilbox#870]
### ISSUE TYPE

 - Bug Report

### Checklist

* [x] `.env` file is attached
* [x] `./check-config.sh` output is added below
* [x] `docker-compose logs` output is added below
* [x] I've looked through the docs: https://devilbox.readthedocs.io/en/latest/
* [x] I've looked through existing issues: https://github.com/cytopia/devilbox/issues
* [x] I've read troubleshooting: https://devilbox.readthedocs.io/en/latest/support/troubleshooting.html

### OS / ENVIRONMENT

1. Host operating system and version: Fedora Linux 35 (Workstation Edition)
2. (Windows only) Native Docker or Docker Toolbox:
3. Docker version: Docker version 20.10.12, build 485636f
4. Docker Compose version: docker-compose version 1.29.2, build unknown
5. (Linux) Is SELinux enabled?: Yes
6. What git commit hash are you on?: v1.10.5

### SUMMARY

I updated devilbox a couple of days ago and I started noticing some queries silently fail returning no results when the IN statement contains at least 1000 parameters.

The problem does not happen using the query in phpMyAdmin but happens using Laravel, I did not try yet using vanilla php.

### STEPS TO REPRODUCE

I'll try to give all the steps later on but I have to try to reproduce it in a minimal enviroment.

In the meanwhile within a Laravel project using:
```
# .env
PHP_SERVER=8.0
HTTPD_SERVER=apache-2.4
MYSQL_SERVER=mariadb-10.5
```
```
#docker-compose.yml
  mysql:
    image: devilbox/mysql:${MYSQL_SERVER}-0.8
OR
    image: devilbox/mysql:${MYSQL_SERVER}-0.15

WORKS WITH compose.override
    image: devilbox/mysql:${MYSQL_SERVER}-0.7
```

Try to run:
```php
dd(App\Models\User::whereIn('id', range(1, 999))->get();)

// This correctly prints the results
//Illuminate\Database\Eloquent\Collection {[#1738 ▼]()
//  #items: array:3 [[▶]()]
//  #escapeWhenCastingToString: false
//}

dd(App\Models\User::whereIn('id', range(1, 1000))->get();)

// This does not print any result
//Illuminate\Database\Eloquent\Collection {[#1733 ▼]()
//  #items: []
//  #escapeWhenCastingToString: false
//}

dd(\App\Models\User::whereIn('id', '>', 0)->whereIn('id', range(1, 999))->get());

// This correctly prints the results again
// to show that the problem is not the number of parameters but only the number of parameters in a single IN.
//Illuminate\Database\Eloquent\Collection {[#1738 ▼]()
//  #items: array:3 [[▶]()]
//  #escapeWhenCastingToString: false
//}
```

### EXPECTED BEHAVIOUR

A query with 1000 parameters in the IN statement in Mysql should return results if they exists.

### ACTUAL BEHAVIOUR

A query with 1000 parameters in the IN statement in Mysq returns no results and no error.

#### Start command

```
$ docker-compose up httpd php mysql
```

#### Outputs

```bash
$ ./check-config.sh

[ERR]   Found 1 error(s) But it's permissions for a directory of another project.
```

```bash
$ docker-compose logs

Last line
`httpd_1  | 2022-03-17 03:41:41,411 INFO success: httpd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)`
```

**Comment by MrCrayon** [comment]
I found this comment on stackoverflow:

> If you have more than 1000 values within the IN() clause MariaDB seems to automatically create temporary tables for performance improvement. You can see this using EXPLAIN

The difference between 10.5.7 and 10.5.8 is not big, but I have no idea if the problem is there or if it could be something else:
https://github.com/MariaDB/server/compare/mariadb-10.5.7...mariadb-10.5.8

**Comment by cytopia** [comment]
@MrCrayon is this a Devilbox issue or an issue specifically with MariaDB?

**Comment by MrCrayon** [comment]
@cytopia it's most probably a MariaDB issue.

I reported it here because I think it would be worth reverting the default to 0.7.

My other goal was to have this public somewhere because it has been really hard to debug so I wanted to share my findings and make life a bit easier to other people.

I do not want to take your time and I think devilbox it's an amazing tool, so feel free to close the issue or tag it as upstream.

**Comment by cytopia** [comment]
What do you think about making it a PR to the troubleshooting section instead? https://devilbox.readthedocs.io/en/latest/support/troubleshooting.html

**Comment by MrCrayon** [comment]
I was not sure how to word this in documentation since I could not find an official bug report but now I found it:
https://jira.mariadb.org/browse/MDEV-27937
 
Affects Version/s 10.3, 10.4, 10.5, 10.6, 10.7, 10.8, 10.9, 10.5.15, 10.6.7
Fix Version/s: 10.3.35, 10.4.25, 10.4.26, 10.5.16, 10.7.4
 
 I was a bit misled because I thought the -08 was connected with MariaDB version but I guess it's a container revision or something like that:
```
07 =>10.5.8-MariaDB-1:10.5.8+maria~focal-log
08 => 10.5.15-MariaDB-1:10.5.15+maria~focal-log
15 => 10.5.15-MariaDB-1:10.5.15+maria~focal-log
```
